public/pages/DetectorConfig/DetectorConfig.scss

```diff
@@ -0,0 +1,69 @@
+.enabled {
+  height: 14.3px;
+  width: 267.91px;
+  color: #454545;
+  font-family: 'Helvetica Neue';
+  font-size: 12px;
+  letter-spacing: 0;
+  line-height: 14px;
+}
+
+.modelSubtitle {
+  height: 17px;
+  width: 162px;
+  color: #3f3f3f;
+  font-family: 'Helvetica Neue';
```

Comment:
Why specify font family? Similar question for other places.
How about we just use default font to keep consistent with other pages?

Reply:
Each component in Lovinda's design has font family specified. Changed to use default font.

Reply:
Sure, thanks, I think the default font is ok.